### ISSUE TITLE
ZeroNet site

### ISSUE THREAD

**Issue by jakefb** [fastcast/fastcast#14]
I found a Fastcast site on ZeroNet with the name Fascast. It works pretty well, I was just wondering who created it? It would be cool to make it dynamic so it uses a database and people can add videos!

http://127.0.0.1:43110/1qJ3EM5rTa2wPTBvJuCFjdLXjrBCgBkR8

**Comment by emuNAND** [comment]
sadly the site hasn't been updated ever since april, but someone could clone it and register a namecoin domain. i am up for the hosting challenge, but i barely know any web development coding.

**Comment by jakefb** [comment]
Hi @emuNAND, thanks for the information! That would be great if you're keen to set it up! It would be cool to create a repository maybe called ZeroFascast with the code and details for the zeronet site so people can find it.

**Comment by emuNAND** [comment]
@jakefb actually, there is a ZeroNetwork Wiki that exists. perhaps i could edit it and lead the dead link to the new one?

**Comment by jakefb** [comment]
@emuNAND, yeah that sounds like a good idea! Since the old site is misspelt as Fascast you should have no problem registering it. Let me know how it goes and I'd love to help with the coding too!

**Comment by emuNAND** [comment]
@jakefb alright, edited! im going to register the namecoin now.

**Comment by jakefb** [comment]
@emuNAND awesome! How's it going? Can you send me the link so I can seed?

**Comment by emuNAND** [comment]
@jakefb it's in the first comment. sadly the blockchain got corrupted so I have to restart.

**Comment by emuNAND** [comment]
added the favicon onto the zite!

**Comment by emuNAND** [comment]
@jakefb how would you feel if we used [this](http://127.0.0.1:43110/1xxooZ7s8iDz3u56i5mNQ1XGmwF8Zvnw9/) as a backup git?

**Comment by jakefb** [comment]
Hi @emuNAND sorry about the really late reply, I've been busy with work. I was able to clone the zeronet git, it's pretty cool so all we need to do is push the project files to it. I have been unable to access http://127.0.0.1:43110/1DzteF7ea3PaaxArGWdh1CLPwQKoB2mrP4/ however it says there are no peers. Thanks for all your help!
